g4f/Provider/selenium/AItianhuSpace.py

```diff
@@ -3,11 +3,11 @@
 import time
 import random
 
-from ..typing import CreateResult, Messages
-from .base_provider import AbstractProvider
-from .helper import format_prompt, get_random_string
-from ..webdriver import WebDriver, WebDriverSession
-from .. import debug
+from ...typing import CreateResult, Messages
+from ..base_provider import AbstractProvider
+from ..helper import format_prompt, get_random_string
+from ...webdriver import WebDriver, WebDriverSession
+from ... import debug
 
 class AItianhuSpace(AbstractProvider):
     url = "https://chat3.aiyunos.top/"
```

Comment:
It's recommended to include a docstring at the beginning of the class to describe its purpose and usage.